jukebox_radio/users/views/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .connect_spotify_view import UserConnectSpotifyView
```

Comment:
Here is a "gotcha" - these have to be imported in the `__init__.py` so they can get picked up by the abbreviated import statements in `urls.py`.

If you were to take these out, then line 4 and 5 of `urls.py` would break saying "no such module found" (or something like that).